AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Subscribe/Support/SortedList.swift

```diff
@@ -20,10 +20,19 @@ class SortedList<ModelType: Model> {
         self.modelSchema = modelSchema
     }
 
+    /// Sets the internal data structures to the list of models. `sortedModels` should already be sorted to efficiently
+    /// update the internal instructures of `SortedList`. If `sortedModels` is not sorted, then the subsequent
+    /// methods used on the data structure will have adverse consequences, for example adding a model assumes the list
+    /// is already sorted (since it performs a binary search insertion).
```

Comment:
what would be an example of adverse consequence? Something acceptable and/or recoverable (like a temporary degrade in performance) or something more severe?

Reply:
the binary search insertion would not accurately find the correct index since the list is no longer sorted. So the items returned will be out of order. This only impacts the caller if they decided to pass in a sort order